packages/teleport/src/components/Empty/Empty.tsx

```diff
@@ -16,29 +16,36 @@ limitations under the License.
 
 import React from 'react';
 
-import { Text, Box, Flex, ButtonPrimary, Link } from 'design';
+import { Text, Box, Flex, ButtonPrimary, ButtonOutlined } from 'design';
 import Card from 'design/Card';
 import Image from 'design/Image';
-import * as Icons from 'design/Icon';
-import empty from './assets';
+
+import application from './assets/appplication.png';
+import database from './assets/database.png';
+import desktop from './assets/desktop.png';
+import stack from './assets/stack.png';
+
+function getAccentImage(entityType: string): string {
```

Comment:
i think it would be better if `entityType` was a literal type, and use the same type for `readOnly.resource`?

Reply:
I added a literal type but I wasn't able to use it for `readOnly.resource` because that is pluralized. I looked into pluralizing the `entityType` value but then I ran into "kubernetes" 😁